lib/rules/metadata/profile.js

```diff
@@ -136,8 +136,19 @@ exports.check = function (sr, done) {
             assembleMeta(id, sr);
         }
     } else {
-        throw new Error(
-            "[EXCEPTION] The document could not be parsed, it's neither a TR document nor a Member Submission."
-        );
+        const getTitle = require('./title').check;
+        let docTitle;
+        await getTitle(sr, (result) => {
+            docTitle = result && result.title;
+        });
+        if (candidate.match("Editor's Draft")) {
```

Comment:
`match` takes a regex as argument. The test won't work here. Also note that candidate is [lower-cased earlier](https://github.com/w3c/specberus/blob/main/lib/rules/metadata/profile.js#L28).
Instead, something like this should work:
```suggestion
        if (candidate.indexOf("editor's draft") > -1) {
```